### ISSUE TITLE
How to classify an image by VGGNet?

### ISSUE THREAD

**Issue by StephenGreat** [tflearn/tflearn#777]
I trained a vgg model using [tflearn/examples/images/vgg_network.py](https://github.com/tflearn/tflearn/blob/master/examples/images/vgg_network.py)

And now I have 
1. model_vgg-430.data-00000-of-00001
2. model_vgg-430.index
3. model_vgg-430.meta

I just want to classify a new image. Thanks a lot!

**Comment by moumine9** [comment]
The best way to do that is to execute the predict functon. Here is an example

~~~

    for i in range(1, len(Xvalid)):
        p = model.predict([ Xvalid[i] ])
        print('Image #{0} | Label = {1} | Prediction = {2} '.format(str(i), str(np.argmax(Yvalid[i])), str(np.argmax(p)) ) ) 
        #print(p)  
~~~

The predict function return an array of each class that you define with your training set, associated with the probabilities to be true. For an example, We are trying to predict the class of an image, predict will return this array [ 0.9  0.2  0.45  0.56  ] . It means there is 90% that the image is class 1,  20% that it is class 2 .... 56% that it is a class 4 images.
The trick that I found, is to return the index of the maximum of the array returned by predict. The index of the maximum run of the Yvalid set is related to the function to_categorical (references here => http://tflearn.org/data_utils/ )

Let me know if you have any questions. ;)

**Comment by StephenGreat** [comment]
Thanks a lot! @moumine9 
I had `model.save()` after `model.fit()` when training. But when predicting is regression layer necessary?
As I know, it needs forward pass when predicting. It seems that regression layer isn't necessary.
I'am new to CNN and tflearn.
